node/tests/utils.rs

```diff
@@ -415,7 +424,17 @@ impl MASQNode {
         config: Option<CommandConfig>,
         remove_database: bool,
     ) -> process::Command {
-        let mut args = Self::minimum_args();
+        let mut args = vec![];
```

Comment:
Since `args` now starts out empty, we don't have to make it mutable and then extend it.